src/sentry/notifications/utils/participants.py

```diff
@@ -383,10 +387,24 @@ def get_send_to(
     event: Event | None = None,
     notification_type: NotificationSettingTypes = NotificationSettingTypes.ISSUE_ALERTS,
     fallthrough_choice: FallthroughChoiceType | None = None,
+    rules: Iterable[Rule] | None = None,
 ) -> Mapping[ExternalProviders, set[RpcActor]]:
     recipients = determine_eligible_recipients(
         project, target_type, target_identifier, event, fallthrough_choice
     )
+
+    if rules:
```

Comment:
In what case would we not have rules?

Reply:
In the typing of rules, it can be None and it fails on a couple tests if there isn't a check for rules (i assume tests where rules is None)

Reply:
Interesting ... I wonder how accurate that is.